bot/exts/utils/snekbox.py

```diff
@@ -390,18 +473,26 @@ async def run_job(
         log.info(f"Received code from {ctx.author} for evaluation:\n{code}")
 
         while True:
-            self.jobs[ctx.author.id] = datetime.datetime.now()
+            self.jobs[ctx.author.id]["has_running_job"] = True
```

Comment:
I would rather use the lock decorator for this. Would result in less state to manually manage. For a custom error message you can use a custom command error handler.

Reply:
Attempted to do this in https://github.com/python-discord/bot/pull/2218/commits/43c6abaeb6047d3566c31db2e0df6f5b42171289 isn't working for some reason though, investigating...

Reply:
It was working, I was just using `int eval` instead of `eval` 🤦